components/resources/speedreader_strings.grdp

```diff
@@ -1,16 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <grit-part>
   <if expr="enable_speedreader">
-    <message name="IDS_SPEEDREADER_BRAND_LABEL" desc="Used in places to show brand title">Brave Speedreader</message>
-    <message name="IDS_SPEEDREADER_BRAND_LABEL_2" desc="Used in places to show brand title"> Speedreader</message>
-    <message name="IDS_SPEEDREADER_ALWAYS_LOAD_LABEL" desc="Used for a checkbox label">Always load speedreader mode for <ph name="HOST_NAME">$1</ph></message>
-    <message name="IDS_SPEEDREADER_THEME_LABEL" desc="Title for changing current page's theme">Theme</message>
-    <message name="IDS_SPEEDREADER_FONT_STYLE_LABEL" desc="Title for changing current page's font style">Font style</message>
-    <message name="IDS_SPEEDREADER_CONTENT_STYLE_LABEL" desc="Title for changing current page's content related properties">Content</message>
-    <message name="IDS_SPEEDREADER_TURN_OFF_DESC" desc="A long form description when speedreader is turned off">
-    Toggle reader mode just for this website. To turn off Speedreader for your entire browser profile, visit <ph name="LINK_BEFORE">$1</ph>Settings.<ph name="LINK_AFTER">$2</ph>
-    </message>
-    <message name="IDS_TOOLTIP_TURN_ON_SPEEDREADER" desc="Tooltip on the turned off speedreader toggle.">Turn ON Speedreader</message>
-    <message name="IDS_TOOLTIP_TURN_OFF_SPEEDREADER" desc="Tooltip on the turned on speedreader toggle.">Turn OFF Speedreader</message>
+    <message name="IDS_READER_MODE_ALWAYS_LOAD_FOR_SITE_LABEL" desc="Used for a checkbox label">Always display in reader mode for this site</message>
+    <message name="IDS_READER_MODE_ALWAYS_LOAD_FOR_ALL_SITES_LABEL" desc="Used for a checkbox label">Always display in reader mode for all sites</message>
```

Comment:
These sound a bit strange to me. Maybe `Always use reader mode ...`. WDYT @rmcfadden3 ?

Reply:
@mkarolin — yeah, I agree. I would suggest we use `reader mode` or else the actual trademarked `speedreader` terminology, as found on https://support.brave.com/hc/en-us/articles/360045031392-What-is-Speedreader- and https://brave.com/features/ …

So I could see either `Always use reader mode` or else `Always use speedreader`. So in this case our strings would either be:

    <message name="IDS_READER_MODE_ALWAYS_LOAD_FOR_SITE_LABEL" desc="Used for a checkbox label">Always use reader mode for this site</message>
    <message name="IDS_READER_MODE_ALWAYS_LOAD_FOR_ALL_SITES_LABEL" desc="Used for a checkbox label">Always use reader mode for all sites</message>

    <message name="IDS_READER_MODE_ALWAYS_LOAD_FOR_SITE_LABEL" desc="Used for a checkbox label">Always use speedreader for this site</message>
    <message name="IDS_READER_MODE_ALWAYS_LOAD_FOR_ALL_SITES_LABEL" desc="Used for a checkbox label">Always use speedreader for all sites</message>

Reply:
Lets go with `Always user speedreader...`

Reply:
@rebron — small typo there in your reply ("user"). You mean `Always [use] speedreader…` yeah?